templates/tools/dockerfile/gcp_api_libraries_venv.include

```diff
@@ -0,0 +1,4 @@
+# Google Cloud Platform API libraries
+# These are needed for uploading test results to BigQuery (e.g. by tools/run_tests scripts)
```

Comment:
Could you add a comment in here which talks about how this file is different from: templates/tools/dockerfile/gcp_api_libraries.include

I know you already added that in the PR conversation, but that would get lost. A comment here would be more persistent.

Reply:
Also it seems like we should prefer this template, and migrate the others to this at some point? Considering this is the new pip way of going it?

Reply:
Added a comment in both the templates. 

> Also it seems like we should prefer this template, and migrate the others to this at some point? Considering this is the new pip way of going it?

Yes, we would need to use virtual envs is the base image has pip 1.23+ which was launched in Jan 2023.